### ISSUE TITLE
Update GB prepare command to handle patch releases

### ISSUE THREAD

**Issue by jhnstn** [wordpress-mobile/release-toolkit-gutenberg-mobile#153]
Blocked by https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/issues/151

Given a patch release the prepare command should:

 - Checkout GB at the prior patch version e.g. if cutting `v1.0.1` the script should checkout GB at the `v1.0.0` tag
 - Accept a set of commits to cherry pick into the patch release branch
 - Proceed with the release preparation

**Comment by jhnstn** [comment]
In theory we should be able to update the change notes programmatically. I would expect that any patch fix would include a change note in the unreleased section. From there cherry picking should filter out any other unreleased change notes... in theory. 

Merging back to trunk will be an issue but that is true regardless of if we automate the change note updates.

**Comment by jhnstn** [comment]
I'm not sure how we want to handle accepting the commits to cherry pick

First, do we want to use PR numbers ? It would be nice to have those for adding to release PRs but we should be able to derive them from the github api.

Also should we have a prompt like the current script? how often do we need to cherry pick multiple commits/PRs ?
We could have the command accept a list of PR numbers (or commits). I think that would be easier if we commonly only cherry pick one or two commits. 

However we'd want to have a confirmation prompt to maker sure the wrangler has requested the correct changes to cherry pick. In this case we should allow for the wrangler to edit the requested changes, at least add more if something was forgotten

I suppose for a first pass we would add the confirm step and if it's not right the wrangler and exit the command and run it again with the correct PR numbers (or commits)
